examples/pulse_control/1_Bringup_experiments.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Given this will cost a customer $40.20 - can we please add a note at the very top of the notebook telling customers how much this will cost. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>